docs/developer-docs/latest/plugins/i18n.md

```diff
@@ -338,9 +338,18 @@ This request:
 
 ### Updating an entry
 
-It is not possible to change the locale of an existing localized entry.
+To update an existing localized entry, send a `POST` request to the appropriate URL depending on the type of content:
 
-When updating a localized entry (with `PUT /{localized-content-type}/:id`), if you set a `locale` attribute in the request body, it will be ignored.
+| Content-Type    | Request URL format                           |
+| --------------- | -------------------------------------------- |
+| Collection type | `POST /api/{content-type}/:id/localizations` |
+| Single type     | `POST /api/{content-type}/localizations`     |
```

Comment:
Here as well should be `PUT` and the URL should be `/api/{content-type}/?locale={locale}`. I'm not sure what the best placeholder term is for the `?locale=` part but that's just for illustration purposes